src/ESPAsyncWebServer.h

```diff
@@ -386,21 +388,15 @@ class AsyncWebServerRequest {
     AsyncResponseStream* beginResponseStream(const String& contentType, size_t bufferSize = RESPONSE_STREAM_BUFFER_SIZE) { return beginResponseStream(contentType.c_str(), bufferSize); }
 
 #ifndef ESP8266
-    [[deprecated("Replaced by beginResponse(...)")]]
+    [[deprecated("Replaced by beginResponse(int code, const String& contentType, const uint8_t* content, size_t len, AwsTemplateProcessor callback = nullptr)")]]
 #endif
     AsyncWebServerResponse* beginResponse_P(int code, const String& contentType, const uint8_t* content, size_t len, AwsTemplateProcessor callback = nullptr) {
       return beginResponse(code, contentType.c_str(), content, len, callback);
     }
 #ifndef ESP8266
-    [[deprecated("Replaced by beginResponse(...)")]]
-#endif
-    AsyncWebServerResponse* beginResponse_P(int code, const String& contentType, PGM_P content, AwsTemplateProcessor callback = nullptr) {
-      return beginResponse(code, contentType.c_str(), content, callback);
-    }
```

Comment:
@vortigont : this is problematic for esp8266 I think (it was not going to the implementation using PGM_P`)